runtime/testy/studentCode.py

```diff
@@ -9,10 +9,28 @@ def autonomous_main():
   pass
 
 def teleop_setup():
-  pass
+  uids = Robot._getAllUIDs()
```

Comment:
So is having "testing" teleop code something we want to leave on master, or should we just have a specific branch for this? Also, as above, (still need to test) but we should be able to auto subscribe.